scraped_data.csv

```diff
@@ -0,0 +1,10 @@
+content
+"Home
```

Comment:
remove these files as they already added to the files folder

Reply:
@sunny7899 which files....?

Reply:
@SparshKhanna0001  
<img width="943" alt="image" src="https://github.com/user-attachments/assets/8abd8f04-ea56-45a2-aeca-78e54f946936">